Week2/todo_app/program.js

```diff
@@ -62,11 +110,61 @@ dbConnection.connect(function(err) {
     console.log('connected as id ' + dbConnection.threadId);
 
     const todoModel = new TodoModel(dbConnection);
+    
+        // Load
     todoModel.load(function(err, todoItems) {
         if(err) {
             console.log("error loading TODO items:", err);
         }
 
         console.log("existing todo items:", todoItems);
     });
+
+        // Create
+    todoModel.create("Teach class 14",1,function(err,todoItems){
```

Comment:
Let's say we'd like to make this app interactive - get input from users/terminal and test all the functionalities you just made. Can you think about how to make this happen?